src/Pickles/Pickles.BaseDhtmlFiles/Index.html

```diff
@@ -14,7 +14,7 @@
     <!--[if lt IE 9]>
       <script src="js/html5.js"></script>
     <![endif]-->
-    #### EMBED EXPERIMENTALS ####
```

Comment:
It's probably best to replaced this with `<!-- EMBED EXPERIMENTALS -->` (there's a corresponding place in the c# code that needs to be replaced as well)